lib/dai-plugin-mcd/src/CdpType.js

```diff
@@ -95,4 +101,37 @@ export default class CdpType {
     const contract = 'PIP_' + this.currency.symbol;
     return this._systemData.get('smartContract').getContractAddress(contract);
   }
+
+  async _getPar() {
+    if (!this._par) this._par = await this._systemData.spot.par();
+    return this._par;
+  }
+
+  async _getVat() {
```

Comment:
`_getVatInfo`, etc. would read better to me -- again, to disambiguate between the whole contract and a piece of data within it